resources/problem.scss

```
@@ -151,6 +157,14 @@ ul.problem-list {
     color: orange;
 }
 
+.has-editorial-color {
```

Can avoid the extra class since you're already tagging with an attribute

```suggestion
td[editorial=1] i {
```

---

But w/e, the rest of the code is written like this already so sure